share/pot/mount-in.sh

```diff
@@ -243,6 +243,10 @@ pot-mount-in()
 		_error "/ is not a valid mount point"
 		return 1
 	fi
+	if _contains_spaces "$_mnt_p" ; then
```

Comment:
Maybe put this check above the "is_absolute_path" check (more generic validation first)?

Reply:
moving the check at earlier stage